runtime/mandala/src/lib.rs

```diff
@@ -1849,122 +1849,122 @@ construct_runtime! {
-		Recovery: pallet_recovery::{Pallet, Call, Storage, Event<T>} = 32,
-		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 33,
-		IdleScheduler: module_idle_scheduler::{Pallet, Call, Storage, Event<T>} = 34,
+		Utility: pallet_utility = 30,
+		Multisig: pallet_multisig = 31,
+		Recovery: pallet_recovery = 32,
+		Proxy: pallet_proxy = 33,
+		IdleScheduler: module_idle_scheduler = 34,
 
 		Indices: pallet_indices::{Pallet, Call, Storage, Config<T>, Event<T>} = 40,
 
 		// Governance
-		GeneralCouncil: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 50,
-		GeneralCouncilMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 51,
-		FinancialCouncil: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 52,
-		FinancialCouncilMembership: pallet_membership::<Instance2>::{Pallet, Call, Storage, Event<T>, Config<T>} = 53,
-		HomaCouncil: pallet_collective::<Instance3>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 54,
-		HomaCouncilMembership: pallet_membership::<Instance3>::{Pallet, Call, Storage, Event<T>, Config<T>} = 55,
-		TechnicalCommittee: pallet_collective::<Instance4>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 56,
-		TechnicalCommitteeMembership: pallet_membership::<Instance4>::{Pallet, Call, Storage, Event<T>, Config<T>} = 57,
-
-		Authority: orml_authority::{Pallet, Call, Storage, Event<T>, Origin<T>} = 70,
-		PhragmenElection: pallet_elections_phragmen::{Pallet, Call, Storage, Event<T>} = 71,
-		Democracy: pallet_democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 72,
+		GeneralCouncil: pallet_collective::<Instance1> = 50,
+		GeneralCouncilMembership: pallet_membership::<Instance1> = 51,
+		FinancialCouncil: pallet_collective::<Instance2> = 52,
+		FinancialCouncilMembership: pallet_membership::<Instance2> = 53,
+		HomaCouncil: pallet_collective::<Instance3> = 54,
+		HomaCouncilMembership: pallet_membership::<Instance3> = 55,
+		TechnicalCommittee: pallet_collective::<Instance4> = 56,
+		TechnicalCommitteeMembership: pallet_membership::<Instance4> = 57,
+
+		Authority: orml_authority = 70,
+		PhragmenElection: pallet_elections_phragmen = 71,
+		Democracy: pallet_democracy = 72,
 
 		// Oracle
 		//
 		// NOTE: OperatorMembership must be placed after Oracle or else will have race condition on initialization
-		AcalaOracle: orml_oracle::<Instance1>::{Pallet, Storage, Call, Event<T>} = 80,
-		OperatorMembershipAcala: pallet_membership::<Instance5>::{Pallet, Call, Storage, Event<T>, Config<T>} = 82,
+		AcalaOracle: orml_oracle::<Instance1> = 80,
+		OperatorMembershipAcala: pallet_membership::<Instance5> = 82,
 
 		// ORML Core
-		Auction: orml_auction::{Pallet, Storage, Call, Event<T>} = 100,
-		Rewards: orml_rewards::{Pallet, Storage, Call} = 101,
-		OrmlNFT: orml_nft::{Pallet, Storage, Config<T>} = 102,
+		Auction: orml_auction  = 100,
```

Comment:
looks like we have some duplicated spaces here

Reply:
Checked and fixed spacing errors.